src/mmw/js/src/modeling/models.js

```diff
@@ -15,9 +15,75 @@ var ResultCollection = Backbone.Collection.extend({
     model: ResultModel
 });
 
-var ProjectModel = Backbone.Model.extend({});
+var ProjectModel = Backbone.Model.extend({
+    urlRoot: '/api/modeling/projects/',
+
+    initialize: function() {
+        // TODO: For a new project, users will eventually
+        // be able to choose which modeling package
+        // they want to use in their project. For
+        // now, the only option is TR55, so it is
+        // hard-coded here. This will likely be a model
+        // itself in the future, with more modification
+        // and input control data.
+        this.set('model_package', 'tr-55');
+        this.set('active_scenario_slug', 'current-conditions');
+        this.set('taskModel', new Tr55TaskModel());
+    },
+
+    saveAll: function() {
+        var self = this;
+
+        // If we haven't saved the project before,
+        // hold on to the scenarios so that we can
+        // save each one once we have an ID for the project
+        if (this.isNew) {
+            var tempScenarios = this.get('scenarios');
+        }
```

Comment:
Having project/scenario endpoints is useful for more atomic updates when editing (changing a modification on a single scenario), which is more common than an initial project save of existing scenarios.  Rather than having multiple types of endpoints to maintain, I kind of like having just the flat `/projects/` and `/scenarios/` endpoints to handle both use cases.